db/seeds/seeds_0.rb

```diff
@@ -0,0 +1,149 @@
+
+require 'faker'
```

Comment:
I don't think you need to require the gem here, I believe it is available if you add it to the Gemfile.